website/src/pages/roadmap.md

```diff
@@ -0,0 +1,338 @@
+
+**Long term, the MACI team’s vision is to build the most secure e-voting solution in the world.**
+
+Short term, our vision is to empower developers to build on MACI to create privacy-protecting voting applications, including quadratic funding platforms & DAO governance tools. Our goal is to provide an out-of-the-box solution for developers to quickly deploy & plug their applications into. MACI enables you to focus less time worrying about voting infrastructure & more time building the logic specific to your application.
+
+### High-level MACI ecosystem goals
+
+1. ~~2023: MACI integrated by 1 PSE project~~ (✅, [QFI](https://github.com/privacy-scaling-explorations/qf))
+2. 2024: MACI integrated by 1 external project
+3. 2025: MACI used by several DAOs for governance & multiple QF integrations
+4. 2028: MACI used in smaller municipal elections (counties, provinces, etc)
+5. 2030: MACI used in national elections
+
+### High-level MACI development goals
+
+1. ~~2023: build technical feasibility for 10K concurrent voters on the platform.~~ (✅)
+2. 2024 Q1: release MACI v1.2, with revamped documentation & educational resources
+3. 2026: build technical feasibility for 100K concurrent voters on the platform.
+4. 2028: build technical feasibility for 1M concurrent voters on the platform.
+
+## QF mission & vision
+
+We’re building technical infrastructure, operational expertise & a network of communities to create a scalable way to enable community organizers to run QF rounds. The mission of our QF project has been to serve as a reference implementation on how to integrate MACI as well as support the practical adoption of this community-funding technology.
+
+**Long term, our vision is to build the most widely adopted QF solution in the world, powered by MACI.**
+
+Our vision to eventually enable QF rounds, running e.g. every quarter, at the local, municipal, national and global scale. We want QF rounds to become the “default” place where projects are contributing funds to public goods
+
+### High-level QF ecosystem goals
+
+1. ~~2023: facilitate 15 QF/QV rounds~~ (❌, 6 rounds)
+2. 2024: facilitate 12 QF/QV rounds
+3. 2025 & beyond: TBD
+
+### High-level QF development goals
+
+1. ~~2023: build QFI as a QV reference implementation for MACI v1.x~~ (✅❓)
+2. 2024: build out reference implementation with complete QF functionality (vs. only QV)
+3. 2024/5: implement additional functionality (key-switching, gas-less voting)
+4. 2025: support additional QF mechanisms
+5. 2025 & beyond: TBD
```

Comment:
Is https://maci.pse.dev/ intended to serve as a team website? If not, I suggest that sections pertaining to QF should ideally be hosted on https://github.com/privacy-scaling-explorations/qf/ instead

Reply:
> Is https://maci.pse.dev/ intended to serve as a team website

Yes, at least for now.

> I suggest that sections pertaining to QF should ideally be hosted on https://github.com/privacy-scaling-explorations/qf/ instead

I agree. I think eventually we should/will move things there. But given how small the team is & we don't have a solid website for QF, I'd like to keep things together for simplicity.

Reply:
@samajammin @baumstern is this conversation resolved?